runtime/compiler/optimizer/UnsafeFastPath.cpp

```diff
@@ -507,11 +507,11 @@ int32_t TR_UnsafeFastPath::perform()
             case TR::sun_misc_Unsafe_getObject_jlObjectJ_jlObject:
                switch (methodSymbol->getRecognizedMethod())
                   {
-               case TR::java_util_concurrent_ConcurrentHashMap_tabAt:
-               case TR::java_util_concurrent_ConcurrentHashMap_setTabAt:
-                  isArrayOperation = true;
-               default:
-                  break;
+                  case TR::java_util_concurrent_ConcurrentHashMap_tabAt:
+                  case TR::java_util_concurrent_ConcurrentHashMap_setTabAt:
+                     isArrayOperation = true;
```

Comment:
As long as you're modifying this file, may I ask you to add a `break` at the end of this case?  There doesn't seem to be any reason for it to want to fall through, and if someone adds another case to this switch in the future, they might not notice they'll need to add a `break`.

Reply:
I can do this. The commit title will also be updated and the commits squashed.